### PR TITLE
[CSFS-111] Develop a Customer details list view page

### PR DESCRIPTION
**Pull request by kojusola**

### CODE REVIEW

File: submissions/adeolaFafemi/phpDatabases/connection.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+try{
+    $pdo = new PDO("mysql:host=localhost;dbname=under9K_store", "root", "nickiminaj_20");
+    
+    $pdo -> setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    $pdo -> getAttribute(constant("PDO::ATTR_CONNECTION_STATUS"));
+    return $pdo;
+    }catch(PDOException $e){
```
Comment (ToluAkin) [inline submissions/adeolaFafemi/phpDatabases/connection.php:9]: Format this block of code @kojusola

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/adeolaFafemi/phpDatabases/connection.php (hunk 1 above)
Comment (kojusola) [reply submissions/adeolaFafemi/phpDatabases/connection.php:8]: Done 
Thank you

Comment (adaugochi) [reply submissions/adeolaFafemi/phpDatabases/connection.php:8]: From line 3 to 10 isn't formated properly

File: submissions/adeolaFafemi/phpDatabases/customerList.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+if(isset($_COOKIE['customers'])){
+   $allData = unserialize($_COOKIE['customers']);
+ }else{
+    $sql = "SELECT * FROM customers";
+    $allData = $pdo->query($sql)->fetchAll();
+    setcookie('customers', serialize($allData), time() + 3600);
+ }
+//  setcookie('customers',  '', time() -60);
+// var_dump (serialize($allData));
+// var_dump ($allData);
```
Comment (adaugochi) [inline submissions/adeolaFafemi/phpDatabases/customerList.php:14]: Remove comments

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adeolaFafemi/phpDatabases/customerList.php (hunk 2 above)
Comment (kojusola) [reply submissions/adeolaFafemi/phpDatabases/customerList.php]: Done 
Thank you

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**